panther_analysis_tool/main.py

```diff
@@ -573,9 +573,18 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
     """
     logging.info("Testing analysis packs in %s\n", args.path)
 
+    search_directories = [args.path]
+
+    # Try the parent directory as well
+    for directory in (HELPERS_LOCATION, '.' + HELPERS_LOCATION,
```

Comment:
Are there cases that scanning the parent directory can cause issues, e.g. conflicts?